BuildScript/Main/Remotion.Local.build

```diff
@@ -35,7 +35,6 @@
     <SkipSourceLinks>True</SkipSourceLinks>
     <SkipNuGet>True</SkipNuGet>
     <SkipDependDB>True</SkipDependDB>
-    <DisableDocker>True</DisableDocker>
```

Comment:
[ ] Don't we have to fix Remotion.Local.build and Remotion.Server.build in Main, too?